src/views/Bond/components/BondModal/components/BondInputArea/BondInputArea.tsx

```diff
@@ -85,46 +91,56 @@ export const BondInputArea: React.VFC<{ bond: Bond; slippage: string; recipientA
   return (
     <Box display="flex" flexDirection="column">
       <WalletConnectedGuard message="Please connect your wallet to purchase bonds">
-        <form onSubmit={handleSubmit}>
```

Comment:
you're right. I had removed it for simplicity sake, since it's only a single input 'form', but to support submitting using the enter key we need to add it back. 
I'm adding support to the input wrapper component for buttonType=submit . This will allow us to use InputWrapper everywhere and have a consistent implementation for styling (form/button side by side on desktop and stacked on mobile), without having to copy paste layout.